src/redux/types.test.ts

```diff
@@ -0,0 +1,138 @@
+import { Status, CourseMeta, Course } from "./types";
+import { CoursesSliceState, FetchCourseResponse } from "./courses/types";
+import { Lesson, Lessons, LessonsSliceState, FetchLessonsParams } from "./lessons/types";
+
+const CourseMetaMock = {
+    slug: "slug",
+    skills: ["skill1", "skill2"],
+    fullCourseProductId: "666",
+    fullCourseProductFamily: "hell",
+    courseVideoPreview: {
+        link: "link",
+        duration: 777,
+        previewImageLink: "link",
+    }
+}
+
```

Comment:
It is a good practice to write tests for your types in TypeScript. By writing tests for your types, you can catch potential issues and ensure that your types are being used correctly throughout your codebase. It also helps to ensure that your types are well-defined and cover all possible scenarios. GJ!

Reply:
I agree)